### PR TITLE
【fix】cache Asynchronous transmission

### PR DESCRIPTION
**Pull request by Mixficsol**

### CODE REVIEW

File: src/pika_cache_load_thread.cc [hunk 1]
```diff
@@ -57,23 +56,19 @@ void PikaCacheLoadThread::Push(const char key_type, std::string &key) {
...
-  slot->DbRWLockReader();
   rocksdb::Status s = slot->db()->GetWithTTL(key, &value, &ttl);
-  slot->DbRWUnLock();
   if (!s.ok()) {
     LOG(WARNING) << "load kv failed, key=" << key;
     return false;
   }
-
+  std::string CachePrefixKeyk = PCacheKeyPrefixK + key;
   cache_->WriteKvToCache(key, value, ttl);
```
Comment (chejinge) [inline src/pika_cache_load_thread.cc:65]: key-> PCacheKeyPrefixK

File: src/pika_cache_load_thread.cc [hunk 2]
```diff
@@ -82,48 +77,40 @@ bool PikaCacheLoadThread::LoadHash(std::string &key, const std::shared_ptr<Slot>
...
-  slot->DbRWLockReader();
   rocksdb::Status s = slot->db()->HGetallWithTTL(key, &fvs, &ttl);
-  slot->DbRWUnLock();
   if (!s.ok()) {
     LOG(WARNING) << "load hash failed, key=" << key;
     return false;
   }
-
+  std::string CachePrefixKeyh = PCacheKeyPrefixH + key;
   cache_->WriteHashToCache(key, fvs, ttl);
```
Comment (chejinge) [inline src/pika_cache_load_thread.cc:86]: key-> PCacheKeyPrefixH

File: src/pika_cache_load_thread.cc [hunk 3]
```diff
@@ -132,14 +119,12 @@ bool PikaCacheLoadThread::LoadSet(std::string &key, const std::shared_ptr<Slot>&
...
-  slot->DbRWLockReader();
   rocksdb::Status s = slot->db()->SMembersWithTTL(key, &values, &ttl);
-  slot->DbRWLockWriter();
   if (!s.ok()) {
     LOG(WARNING) << "load set failed, key=" << key;
     return false;
   }
-
+  std::string CachePrefixKeys = PCacheKeyPrefixS + key;
   cache_->WriteSetToCache(key, values, ttl);
```
Comment (chejinge) [inline src/pika_cache_load_thread.cc:128]: 1

File: src/pika_cache_load_thread.cc [hunk 4]
```diff
@@ -172,22 +155,19 @@ bool PikaCacheLoadThread::LoadZset(std::string &key, const std::shared_ptr<Slot>
...
-  slot->DbRWLockReader();
   rocksdb::Status s = slot->db()->ZRangeWithTTL(key, start_index, stop_index, &score_members, &ttl);
-  slot->DbRWUnLock();
   if (!s.ok()) {
     LOG(WARNING) << "load zset failed, key=" << key;
     return false;
   }
-
+  std::string CachePrefixKeyz = PCacheKeyPrefixZ + key;
   cache_->WriteZSetToCache(key, score_members, ttl);
```
Comment (chejinge) [inline src/pika_cache_load_thread.cc:164]: key-> PCacheKeyPrefixZ

File: src/pika_cache_load_thread.cc (hunk 1 above)
Comment (Mixficsol) [reply src/pika_cache_load_thread.cc]: done

File: src/pika_cache_load_thread.cc (hunk 2 above)
Comment (Mixficsol) [reply src/pika_cache_load_thread.cc]: done

File: src/pika_cache_load_thread.cc (hunk 4 above)
Comment (Mixficsol) [reply src/pika_cache_load_thread.cc]: done

**Review by chejinge — APPROVED** [review]

**Pull request merged**